src/lib/ThumbnailsSidebar.js

```diff
@@ -311,7 +311,7 @@ class ThumbnailsSidebar {
      * @return {Promise} - promise reolves with the image HTMLElement or null if generation is in progress
      */
     createThumbnailImage(itemIndex) {
-        const cacheEntry = this.thumbnailImageCache.get(itemIndex);
+        const cacheEntry = this.thumbnailImageCache && this.thumbnailImageCache.get(itemIndex);
```

Comment:
> This change seems unrelated to the archive viewer. Is it meant to address a separate bug?

This change is to fix a sanity test. The `thumbnailImageCache` is sometimes not initialized in time because the travis testing machine is too slow. Should I open another pr for this?

Reply:
Should we revert this change since we have the other PR open?